backend/src/main/resources/templates/Rhosak/actionRequiredTitle.txt

```diff
@@ -1 +1 @@
-{action.context.system_check_in.toUtcFormat()} - {action.events.size()} {#if action.events.size() is 1} Action {#else} Actions {/if} Required on {action.context.display_name}
+{action.context.system_check_in.toUtcFormat()} - This notification is for your {action.events.size()} Kafka {#if action.events.size() is 1} instance {#else} instances{/if}
```

Comment:
This will the notification/email title, right?

If so, it should mention `Action Required`.

Reply:
Good catch. It is missing Action Required in the beginning.

Reply:
Thanks Manyanda